tests/test_file_util.py

```diff
@@ -0,0 +1,54 @@
+from unittest.mock import MagicMock, call
+
+from google.cloud import storage
+
+from wdl_runner.file_util import gcs_cp
+
+
+def test_gcs_cp():
```

Comment:
debating if to leave as is or change to actually upload the files to GCS (might make the test clearer)

Reply:
i think mocking is good enough :) We can do the actual testing via integration testing!